### DISCUSSION TITLE
Lockfile maintenance approval

### DISCUSSION THREAD

**Discussion by kfiven** [renovatebot/renovate#18362] [Request Help] [answered]
Hi, I have enabled Dependency dashboard approval for Renovate bot but it still keep ignoring that in case of Lockfile maintanence. My config file is https://github.com/cinnyapp/cinny/blob/dev/.github/renovate.json

How can make it follow the approval for lockfile maintenance also?

Thanks!

**Comment by rarkins** [comment, marked as answer]
Make sure your lock file maintenance branch is deleted. Approval controls the creation of the branch, so if the branch already exists then approval does nothing.

https://github.com/cinnyapp/cinny/blob/renovate/lock-file-maintenance/.github/renovate.json
